### PR TITLE
include doorway environment in doorway GET

### PR DESCRIPTION
**Pull request by guscost**
Should fix DASH-585 with no backend changes needed.

### CODE REVIEW

File: src/rx-actions/route-transition/admin-brivo-mappings.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ async function fetchAllBrivoSites(dispatch) {
   try {
     // BRIAN: Not sure how guys handle loading states, but I'm using this to determine if it's loading...
     dispatch({type: 'BRIVO_SITES_LOADING', sitesLoading: true})
-    doorways = await fetchAllObjects<DensityDoorway>('/doorways', { cache: false });
+    doorways = await fetchAllObjects<DensityDoorway>('/doorways?environment=true', { cache: false });
     doorwayMappings = await fetchAllObjects<DensityDoorway>('/integrations/doorway_mappings/', { cache: false });
```
Comment (jshanley) [inline src/rx-actions/route-transition/admin-brivo-mappings.ts]: doorway mappings are also type `DensityDoorway`?

File: src/rx-actions/route-transition/admin-locations-edit.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export async function loadData(dispatch) {
     [hierarchy, spaces, doorways, labels, tags, assignedTeams] = await Promise.all([
       await fetchAllObjects<DensitySpaceHierarchyItem>('/spaces/hierarchy', { cache: false }),
       await fetchAllObjects<DensitySpace>('/spaces', { cache: false }),
-      await fetchAllObjects<DensityDoorway>('/doorways', { cache: false }),
+      await fetchAllObjects<DensityDoorway>('/doorways?environment=true', { cache: false }),
```
Comment (jshanley) [inline src/rx-actions/route-transition/admin-locations-edit.ts]: what does this query param do?

Comment (jshanley) [reply src/rx-actions/route-transition/admin-locations-edit.ts]: oh that's the whole point ;)

Comment (jshanley) [reply src/rx-actions/route-transition/admin-locations-edit.ts]: Looking at the implementation of `fetchAllObjects` it looks like you can pass the additional query parameter like this:

``` typescript
fetchAllObjects<DensityDoorway>('/doorways', {
  cache: false,
  params: {
    environment: 'true',
  }
})
```

File: src/rx-actions/route-transition/admin-brivo-mappings.ts (hunk 1 above)
Comment (guscost) [reply src/rx-actions/route-transition/admin-brivo-mappings.ts]: environment mappings? yes those are part of the typedef

File: src/rx-actions/route-transition/admin-locations-edit.ts (hunk 2 above)
Comment (guscost) [reply src/rx-actions/route-transition/admin-locations-edit.ts]: nice, thanks, will change

File: src/rx-actions/route-transition/admin-brivo-mappings.ts (hunk 1 above)
Comment (guscost) [reply src/rx-actions/route-transition/admin-brivo-mappings.ts]: oh, no idea about doorway mappings... it probably turns into an `any` type implicitly anyway

**Pull request merged**